webpack/development.config.js

```diff
@@ -117,7 +117,7 @@ const baseDevConfig = () => ({
       {
         test: /\.inline\.svg$/,
         issuer: /\.js$/,
-        loader: 'svg-inline-loader?removeSVGTagAttrs=false&removeTags=true&removingTags=title&removingTags=desc&idPrefix=[sha512:hash:hex:5]-',
+        loader: 'svg-inline-loader?removeSVGTagAttrs=false&removeTags=true&removingTags[]=title&removingTags[]=desc&idPrefix=[sha512:hash:hex:5]-',
```

Comment:
```suggestion
        loader: 'svg-inline-loader?removeSVGTagAttrs=false&removeTags=true&removingTags=["title", "desc"]&idPrefix=[sha512:hash:hex:5]-',
```

Reply:
Looking at their tests, I think that should work too, but up to you. I don’t think is that important.

Reply:
Nope. I tried this originally and it didn't work but this other format does :(